admin/notifications/views.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.decorators import user_passes_test
+from django.shortcuts import render, redirect
+from admin.base.utils import osf_staff_check
+from osf.models.notifications import NotificationSubscription
+from django.db.models import Count
+
+@user_passes_test(osf_staff_check)
+def handle_duplicate_notifications(request):
+    duplicates = NotificationSubscription.objects.values('user', 'node', 'event_name').annotate(count=Count('id')).filter(count__gt=1)
```

Comment:
It's typically a linting error to have a line longer then 100 characters, but I don't think it's enforced in the admin app, try to keep that standard anyway and break up lines over 100 characters.